### PR TITLE
Add TSLA Stock Command

### PR DESCRIPTION
**Pull request by sinnafuls**

### CODE REVIEW

**Comment by sttephen** [comment]
Can we remove the wix stock command and add this instead? Not a fan of having >1 of these commands. It is clunky but it is definitely useful for pinging the bot and also SEEING HOW MUCH MONEY I LOSE EVERY DAY!!!

**Comment by Nuckerr** [comment]
Would prefer to keep it at six bcs it's funny and bears more meaning. I mean this isn't a stock bot

**Comment by sinnafuls** [comment]
TSLA > WIX

Useful command to see how much money me and stephen loose every day, replace one meme command with a better meme command 😎

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Learnspigot Discord bot
-The discord bot for the [Learnspigot Discord server](https://learnspigot.com/discord). This powers everything from verification to reputation in help forums and checking the stock of WIX.  
+# LearnSpigot Discord Bot
```
Comment (Nuckerr) [inline README.md]: This should be in its own pr. Try to not add multiple features in one pr

**Review by Nuckerr — CHANGES_REQUESTED** [review]
Remove the wix command. Also is this tested?

**Comment by sttephen** [comment]
@Nuckerr I am pretty sure it has not been tested.

File: README.md (hunk 1 above)
Comment (sinnafuls) [reply README.md]: It is indeed tested

**Comment by sinnafuls** [comment]
Removed WixStockCommand, TslaStockCommand is tested and works.

**Review by Nuckerr — APPROVED** [review]
